phoenicis-dist/src/flatpak/org.phoenicis.javafx.json

```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "org.phoenicis.javafx",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "add-extensions": {
```

Comment:
I think there should be extension for 32bit libGL `org.freedesktop.Platform.GL32`. Without it, 32bit wine may be unable to use NVidia libGL.
See how [steam flatpak](https://github.com/flathub/com.valvesoftware.Steam/blob/47015a67c4a79a7dc98d29f79c31f00388a9801b/com.valvesoftware.Steam.json#L36) adds 32bit extensions.

Reply:
I've actually tried that but I always got an error when I tried to `flatpak install` this. Maybe I just did it wrong. Can you try?